js/MapManager.js

```diff
@@ -281,23 +281,16 @@ var MapManager = (function($, d3, leaflet) {
 
       uniqueLocs.forEach(function(item) {
          // setTimeout(function() {
-          if (item.className == "campaign-office") {
+          if (item.className == "meet-senator-turner"){
             L.marker(item.latLng, {icon: CAMPAIGN_OFFICE_ICON, className: item.className})
```

Comment:
Need to change `CAMPAIGN_OFFICE_ICON`?

Reply:
We're using same icon, so didn't rewrite references in code. CAMPAIGN_OFFICE_ICON is just a star. Wanted to touch as little lines of code as I could because everything is so brittle.